### PR TITLE
Added dcos task metrics summary/details in Go.

### PR DESCRIPTION
**Pull request by armandgrillet**

### CODE REVIEW

File: pkg/cmd/task/task_metrics_summary.go [hunk 1]
```diff
@@ -13,6 +20,73 @@ func newCmdTaskMetricsSummary(ctx api.Context) *cobra.Command {
...
+			if cpuTotal == 0 {
+				cpuTotal = math.MaxFloat64
+			}
+			memUsed := summaryDatapoints["mem.total_bytes"]
+			memTotal := summaryDatapoints["mem.limit_bytes"]
+			if memTotal == 0 {
+				memTotal = math.MaxFloat64
+			}
+			diskUsed := summaryDatapoints["disk.limit_bytes"]
+			diskTotal := summaryDatapoints["disk.used_bytes"]
```
Comment (bamarni) [inline pkg/cmd/task/task_metrics_summary.go]: Total and used seem shifted?

File: pkg/cmd/task/task_metrics_summary.go [hunk 2]
```diff
@@ -13,6 +20,73 @@ func newCmdTaskMetricsSummary(ctx api.Context) *cobra.Command {
...
+
+			table := cli.NewTable(ctx.Out(), []string{"CPU", "MEM", "DISK"})
+			// This value mapping does not make sense (especially for mem and disk) but that is how it was in the Python codebase.
+			cpuUsed := summaryDatapoints["cpus.user_time_secs"] + summaryDatapoints["cpus.system_time_secs"]
+			cpuTotal := cpuUsed + summaryDatapoints["cpus.throttled_time_secs"]
+			if cpuTotal == 0 {
+				cpuTotal = math.MaxFloat64
+			}
+			memUsed := summaryDatapoints["mem.total_bytes"]
+			memTotal := summaryDatapoints["mem.limit_bytes"]
```
Comment (bamarni) [inline pkg/cmd/task/task_metrics_summary.go]: `mem.total_bytes`?

File: pkg/cmd/task/task_metrics_summary.go [hunk 3]
```diff
@@ -13,6 +20,73 @@ func newCmdTaskMetricsSummary(ctx api.Context) *cobra.Command {
...
+			if memTotal == 0 {
+				memTotal = math.MaxFloat64
+			}
+			diskUsed := summaryDatapoints["disk.limit_bytes"]
+			diskTotal := summaryDatapoints["disk.used_bytes"]
+			if diskTotal == 0 {
+				diskTotal = math.MaxFloat64
```
Comment (bamarni) [inline pkg/cmd/task/task_metrics_summary.go]: Shall we print `N/A` here? If the total is missing we can't print a useful percentage.

File: pkg/cmd/task/task_metrics_summary.go (hunk 1 above)
Comment (armandgrillet) [reply pkg/cmd/task/task_metrics_summary.go]: IT's apparently not a bug but a feature: https://github.com/dcos/dcos-core-cli/blob/1b4d93adfb84006d4c676f135ab63d3aaf6b1e70/python/lib/dcoscli/dcoscli/metrics.py#L187

File: pkg/cmd/task/task_metrics_summary.go (hunk 2 above)
Comment (armandgrillet) [reply pkg/cmd/task/task_metrics_summary.go]: Not a bug but a feature: https://github.com/dcos/dcos-core-cli/blob/1b4d93adfb84006d4c676f135ab63d3aaf6b1e70/python/lib/dcoscli/dcoscli/metrics.py#L184

**Pull request merged**